README.md

```diff
@@ -65,6 +65,7 @@ var errors = require('@google/cloud-errors').start({
 	projectId: 'my-project-id',
 	key: 'my-api-key',
 	reportUncaughtExceptions: false, // defaults to true.
+	logLevel: 0, // defaults to logging warnings (2). Available levels: 0-5
```

Comment:
@matthewloring Yeah, I'm just showing you can provide a different value and reporting the default value in the comment like the preceding line.